server/routes/journeys/record-investigation/why-behaviour-occurred/view.njk

```diff
@@ -13,24 +13,17 @@
     </label>
     <form method="post">
         <input type="hidden" name="_csrf" value="{{ csrfToken }}"/>
-        {{ govukDetails({
-            summaryText: "What type of information to include",
```

Comment:
`{% from "govuk/components/details/macro.njk" import govukDetails %}` is unused and can be deleted